src/configmanager.cpp

```diff
@@ -248,6 +248,8 @@ bool ConfigManager::load()
 	boolean[ONLY_INVITED_CAN_MOVE_HOUSE_ITEMS] = getGlobalBoolean(L, "onlyInvitedCanMoveHouseItems", true);
 	boolean[REMOVE_ON_DESPAWN] = getGlobalBoolean(L, "removeOnDespawn", true);
 	boolean[PLAYER_CONSOLE_LOGS] = getGlobalBoolean(L, "showPlayerLogInConsole", true);
+	boolean[MANASHIELD_BREAKABLE] = getGlobalBoolean(L, "useBreakableManaShield", false);
+	boolean[MANASHIELD_BREAKABLE_FOR_ALL] = getGlobalBoolean(L, "manaShieldBarForAll", false);
```

Comment:
what does this setting does?

Reply:
probably toggles between classic and new magic shield

Reply:
> probably toggles between classic and new magic shield

that is the MANASHIELD_BREAKABLE I suppose